lib/workers/repository/process/extract-update.ts

```diff
@@ -166,10 +167,25 @@ export async function extract(
   return packageFiles;
 }
 
+async function fetchVulnerabilities(
```

Comment:
doesn't it makes sense to move this to the imported files

Reply:
Tests for `fetchVulnerabilities` need a lot of mocking that is specific to `extract` and `lookup` in `extract-update.ts`. I'd prefer to rather keep these together in `extract-update.spec.ts` and not replicate them to `vulnerabilities.spec.ts`, as they are irrelevant for testing the functionality of the `Vulnerabilities` class itself.